protocol/Makefile

```diff
@@ -233,11 +233,13 @@ test-unit-and-integration:
 benchmark:
 	@VERSION=$(VERSION) go test -mod=readonly -tags='$(build_tags)' -bench=. ./...
 
-test-container:
-	@SKIP_DISABLED=true VERSION=$(VERSION) go test -mod=readonly -tags='container_test $(build_tags)' ./...
+test-container: # run ./app/upgrades/v5.2.0 first to avoid resource conflict
+	@SKIP_DISABLED=true VERSION=$(VERSION) go test -mod=readonly -tags='container_test $(build_tags)' ./app/upgrades/v5.2.0
+	@SKIP_DISABLED=true VERSION=$(VERSION) go test -mod=readonly -tags='container_test $(build_tags)' $$(go list ./... | grep -v './app/upgrades/v5.2.0')
 
-test-container-accept:
-	@SKIP_DISABLED=true VERSION=$(VERSION) go test -mod=readonly -tags='container_test $(build_tags)' ./... -args -accept
+test-container-accept: # run ./app/upgrades/v5.2.0 first to avoid resource conflict
+	@SKIP_DISABLED=true VERSION=$(VERSION) go test -mod=readonly -tags='container_test $(build_tags)' ./app/upgrades/v5.2.0 -args -accept
+	@SKIP_DISABLED=true VERSION=$(VERSION) go test -mod=readonly -tags='container_test $(build_tags)' $$(go list ./... | grep -v './app/upgrades/v5.2.0') -args -accept
```

Comment:
non-blocking: What was the issue here? This change means that any future upgrade testing would require updating these command as well, so wondering if there was a better solution that specifically specifying the file to run.

Reply:
without this change, `app/upgrades/v5.2.0` and `testing/container` are being run in parallel, leading to resource conflict (attempting to use the same port). This is more of an issue for 5.2 because 5.2's upgrade test takes long as it needs to wait until block 50

in `main`, don't think this change is needed as we won't run 5.2 upgrade test